exercises/strain/strain_test.go

```diff
@@ -19,6 +19,14 @@ import (
 	"testing"
 )
 
+const targetTestVersion = 1
+
+func TestTestVersion(t *testing.T) {
```

Comment:
In #470 I definatly read 

> placement of TestTestVersion function as the first test function

to indicate we were floating it to the top of the test functions (i.e. functions that start with Test).